template.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jonathan work</title>
+    <link href="minimal.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+        <form action="" method="get">
```

Comment:
Formatting needs to be fixed on the entire file again.
Remember: ctrl+a, ctrl+k, ctrl+f